scripts/fetch-esgf.py

```diff
@@ -174,6 +180,7 @@ def fetch(self, remove_ensembles: bool = True):
                 "vegFrac",
             ],
             experiment_id=["historical"],
+            table_id=("Amon", "Emon", "Lmon"),
```

Comment:
This will probably still fetch a bit too much data, see https://github.com/Climate-REF/climate-ref/pull/414, but maybe that's actually nice because it allows testing in a more real world scenario

Reply:
I'm not too worried about making it super tight as long as it isn't ocean/atmos datasets. It doesn't make much of a difference if some of the more esoteric runs are also captured.